src/Microsoft.DotNet.Cli.Utils/EnvironmentProvider.cs

```diff
@@ -47,6 +47,7 @@ private IEnumerable<string> SearchPaths
                         .GetEnvironmentVariable("PATH")
                         .Split(s_pathSeparator)
```

Comment:
Consider StringSplitOptions.RemoveEmptyEntries, though you still need to filter on whitespace I guess for e.g."path1;   ;path2" I guess.